snake_test.go

```diff
@@ -44,18 +44,20 @@ func toSnake(tb testing.TB) {
 		{"ManyManyWords", "many_many_words"},
 		{"manyManyWords", "many_many_words"},
 		{"AnyKind of_string", "any_kind_of_string"},
-		{"numbers2and55with000", "numbers_2_and_55_with_000"},
+		{"numbers2and55With000", "numbers2and55_with000"},
```

Comment:
Here you have changed not only the expected, but also the input - from `numbers2and55with000` to `numbers2and55With000`. Intentional?

Reply:
Changed it back.